core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -98,12 +98,27 @@ public class DateTimeFunction {
   // Map used to determine format output for the get_format function
   private static final Table<String, String, String> formats =
       ImmutableTable.<String, String, String>builder()
-      //TODO: Add support for other formats
-      .put("date", "usa", "%m.%d.%Y")
-      .put("time", "usa", "%h:%i:%s %p")
-      .put("datetime", "usa", "%Y-%m-%d %H.%i.%s")
-      .put("timestamp", "usa", "%Y-%m-%d %H.%i.%s")
-      .build();
+          .put("date", "usa", "%m.%d.%Y")
```

Comment:
Consider making DATE_USA and TIME_USA and then you can re-use it for datetime and tests.

Reply:
Consider adding "usa" as a constant to re-use in tests

Reply:
Consider adding "date" and "datetime", etc into a constant

Reply:
I've decided against doing this for now. When I try to refactor the code, it ends up looking fairly messy because many of the formats have minor differences, so it results in harder-to-read code overall. I've also decided against using the constants and sharing them with the test code because if they are shared between this class and tests, many of the tests could still pass even if the format is wrong.